code/controllers/configuration/entries/general.dm

```diff
@@ -352,6 +352,14 @@
 
 /datum/config_entry/flag/check_randomizer
 
+/datum/config_entry/string/md5_salt
+	protection = CONFIG_ENTRY_HIDDEN
+
+/datum/config_entry/string/md5_salt/ValidateAndSet(str_val)
+	if(str_val == "PLEASE_CHANGE_ME123")
+		WARNING("[name] is using the default salt, please change!")
```

Comment:
You think that'll be effective?

Reply:
If they still somehow fail to do it, they deserve anything bad that coming their way